app/_data/tables/version_errors_konnect.yml

```diff
@@ -1351,4 +1362,15 @@ messages:
     Resolution: |
       Please upgrade Kong Gateway to version `3.2` or above.
     DocumentationURL: |
-      [AWS Lambda plugin](/hub/kong-inc/aws-lambda/)
\ No newline at end of file
+      [AWS Lambda plugin](/hub/kong-inc/aws-lambda/)
+  - ID: P162
```

Comment:
@lena-larionova Does order matter for the rendering of the documentation?

Reply:
It just gets generated in the same order as what's in this file. See:
https://deploy-preview-5163--kongdocs.netlify.app/konnect/runtime-manager/troubleshoot/

Reply:
Ok thank you, sorry about not looking at the rendered components.

I read the PR wrong anyway so this helped a ton.